src/theory/strings/theory_strings.cpp

```diff
@@ -1056,24 +1056,28 @@ void TheoryStrings::checkMemberships()
 {
   // add the memberships
   std::vector<Node> mems = getExtTheory()->getActive(kind::STRING_IN_REGEXP);
+  // maps representatives regular expression memberships for that class
```

Comment:
```suggestion
  // maps representatives to regular expression memberships in that class
```
(the original sentence doesn't quite parse for me)